### ISSUE TITLE
Tabs: Not navigable with arrow keys

### ISSUE THREAD

**Issue by tskarhed** [grafana/grafana#89403]
**Steps to reproduce**:

Go to dashboard settings. Try to navigate between the tabs using arrow keys.

probably related to https://github.com/grafana/grafana/issues/89402

**Actual Result**:

Nothing happens.

**Expected Result**

The focus should be changed between them. Although the exact behaviour is unclear, since these are links. In the non-link case, the corresponding panel should be displayed.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tablist_role

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)

**Environment**:
- Grafana version:
- Data source type & version:
- User OS & Browser:
- Others:

**Comment by tskarhed** [comment]
For this to be valuable, I think we need to create a new Tabs component.

**Comment by L-M-K-B** [comment]
I had a look at this during the hackathon week and it seems a lot more effort than I expected in the first place. The effort is at least medium so I added the `effort/medium` label and put it back to next in order to discuss the approach with the squad again.

**Comment by L-M-K-B** [comment]
This issue will be handled in the following issue summarising all a11y related issues: https://github.com/grafana/grafana/issues/92004
